src/Sarif/Errors.cs

```diff
@@ -528,7 +548,7 @@ private static string GetMessageFormatResourceForNotification(string notificatio
             string resourceName = notificationId.Replace('.', '_');
 
             return (string)typeof(SdkResources)
-                            .GetProperty(resourceName, BindingFlags.NonPublic | BindingFlags.Static)
+                            .GetProperty(resourceName, BindingFlags.Public | BindingFlags.Static)
```

Comment:
>Public [](start = 68, length = 6)

`public` because `DriverUtilities` (in the separate Sarif.Driver assembly) needs to see one of the resource strings.